recipes/ccache/all/conanfile.py

```diff
@@ -1,9 +1,13 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
-import functools
+from conan import ConanFile
+from conan.tools.cmake import cmake_layout, CMake, CMakeToolchain, CMakeDeps
+from conan.tools.gnu import PkgConfigDeps
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import export_conandata_patches, apply_conandata_patches, copy, get
+from conan.tools.build import cross_building, check_min_cppstd
+from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.33.0"
+required_conan_version = ">=1.53.0"
```

Comment:
```suggestion
required_conan_version = ">=1.55.0"
```

https://docs.conan.io/en/latest/reference/conanfile/tools/cmake/cmakedeps.html#set-property